### ISSUE TITLE
Messaging: Replay Recommender

### ISSUE THREAD

**Issue by joethreepwood** [PostHog/meta#253]
# Messaging

The trouble with replays is that it's easy to forget to use them and it's hard to find the signal amidst the noise. 

We've got some solutions to the latter now (activity scores), so here's a solution to the former: An email campaign that tells people who forget that they've got some cool new replays to watch. 

## Workflow

Users will enter this campaign if: 

- They haven't watched a recording in 3 weeks
- They have ingested over 500 recordings

This ensures we are reaching people who have recordings to watch, but aren't watching them. 

Once they're in the flow, we immediately trigger the following email: 

<img width="760" alt="Screenshot 2024-10-29 at 17 01 33" src="https://github.com/user-attachments/assets/c538ceba-224c-4d00-bf53-42d435d00d1d">

If they click a link in that email, we then set `replay_recommender_conversion` to `true` on the person
The link we drive them towards is a list of activity score recordings in the last 24 hours which are greater than 30 seconds. 

They then exit the campaign. They can't enter it again for another 4 weeks. This stops the emails becoming annoying.

## Subscription changes

We want to give users a way to opt-out of these emails _without_ opting out of other emails. So, I've created a new subscription type and updated the existing ones. 

<img width="510" alt="Screenshot 2024-10-29 at 15 31 41" src="https://github.com/user-attachments/assets/b0715c2b-8890-487d-876e-8ae6f396f7ee">

**Comment by joethreepwood** [comment]
@raquelmsmith open to your thoughts here on what's worth adding an experiment for, etc

**Comment by raquelmsmith** [comment]
Nice! Love the hogbot 😍 

I wonder if it's worth telling them about the "what to watch" page? Eg. "Need some other inspiration? Check out our what to watch page." Or could cycle through specific recommendations - "want another idea? how about watching users who used your signup flow?" and then we could highlight the specific template when they reach the page. (I'm okay with it if you think it's a bad idea, but since the goals are similar - help people see the variety of ways they can use replays - it might go well together)

---

An experiment would be good.

Control group does not get the email, prop on person (or event? I'd kind of rather do this with an event) that says they were entered into the experiment with a variant `control`.

Test group does get the email, prop/event saying they were entered into exp with variant `test`.

I'd make the control group 20% of the users and the test 80% (since most prob won't click anyway - though receiving the email at all could remind them that it exists!)

**Comment by joethreepwood** [comment]
> Nice! Love the hogbot 😍
> 
> I wonder if it's worth telling them about the "what to watch" page? Eg. "Need some other inspiration? Check out our what to watch page." Or could cycle through specific recommendations - "want another idea? how about watching users who used your signup flow?"

This is there are the moment, albeit buried in the footer. We can move it out, but I think we still want it minimized -- having two CTAs which are boldly put together is going to complicate the messaging, I think. 

<img width="610" alt="Screenshot 2024-10-30 at 10 43 42" src="https://github.com/user-attachments/assets/b410c257-c89e-45ea-8c3c-a1ac4c04bcf0">

> An experiment would be good.
> 
> Control group does not get the email, prop on person (or event? I'd kind of rather do this with an event)

Do we need an event to do this? I can easily configure the experiment in Customer.io and track success there, without it. I'll set it up now, but the Customer.io integration isn't two-way -- so I don't think this event will make it back to PostHog anyway.

**Comment by joethreepwood** [comment]
All set up. We'll set an event and see what happens with that, but the A/B test in Customer.io should give us a decent read anyway and I can loop back with the results once it's running. 

<img width="674" alt="Screenshot 2024-10-30 at 10 54 54" src="https://github.com/user-attachments/assets/e6f2d0e1-cee7-4b1b-8fa3-fb75f0851259">

Unless any concerns, I'll set this to run tomorrow morning?

**Comment by raquelmsmith** [comment]
🙌 awesome! What's the event that's being sent?

**Comment by joethreepwood** [comment]
> 🙌 awesome! What's the event that's being sent?

`replay_recommender_conversion`

<img width="680" alt="Screenshot 2024-10-30 at 15 59 55" src="https://github.com/user-attachments/assets/7d20a31d-a321-41a4-a9fe-5c63c47d07e6">

**Comment by raquelmsmith** [comment]
Oh is this sending an event/updating the person in customer.io? I meant an event/prop in posthog 😄

**Comment by joethreepwood** [comment]
We hit a snag with this. 

One user replied saying that he didn’t use replays at all. Logged in as him and confirmed that he had no replays at all. Debugged with Annika and we can’t see any real usage from him in the last 12 weeks, no recordings on file. 

We've realized this is due to the `organization_usage_report` segment not having a filter on it. I've now added a filter saying that that event must have come in the last 21 days, which will hone it in. 

All these emails are coming directly from me at the moment, so I'll respond to any users who flag this.
